release/assets/macos_cli_setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+print_help() {
+  cat << EOF
+
+Conjur CLI setup script
+
+Installs and configures the Conjur CLI on macOS environments.
+Upon completion of this setup script, you will be able to access the Conjur CLI from anywhere on your machine.
+The installation creates a symbolic link between the CLI executable and directory path. By default, this path is '/usr/local/bin'.
+To overwrite this path, configure the directory path parameter.
+
+For example ./macos_cli_installation.sh /usr/bin
+
+Usage: ./macos_cli_setup.sh [options]
```

Comment:
Don't have to use script name, name can change cane use $0 in some way.

Reply:
replacing with $0